webapp/components/login/login.jsx

```diff
@@ -35,6 +35,8 @@ export default class Login extends React.Component {
         this.state = state;
     }
     componentDidMount() {
+        document.title = global.window.mm_config.SiteName;
```

Comment:
The title is already set from root.html (which is almost the same as setting in root.jsx), then the channel loads and re-sets the title.  Then when logging out you end up at /login with the wrong title, but root.jsx is still mounted so the title never changes.  The /login page needs to override the title.